<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_produto_cadastro</class>
 <widget class="QMainWindow" name="Tela_produto_cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>837</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item alignment="Qt::AlignLeft">
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background: transparent;
border: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../imagem/icons/back.png</normaloff>../imagem/icons/back.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>40</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>75</width>
            <height>41</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background: transparent;
border: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/power-off.png</normaloff>icons/power-off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>90</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../imagem/MOTORMAX-LGO.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="hearder">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>890</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
font-size: 22px;
margin-top: 30px;
margin-bottom: 15px;
font: plain;
}</string>
         </property>
         <property name="text">
          <string>Cadastrar novo Produto</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="card">
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>500</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 240);
border-radius: 15px;
border: 1px solid rgba(0, 0, 0,0.1); </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>360</y>
         <width>150</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #f59e0b;
color: black;
border: none;
border-radius: 8px;
padding: 12px;
font-size: 14px;
font-weight: bold;
min-height: 20px;
}

QPushButton:hover{
background-color: #d97706;
}

QPushButton:pressed{
background-color: #b45309;
}

QPushButton:disable{
background-color: #582503;
}</string>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Pre√ßo *</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>311</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
border: 2px solid #e5e7eb;
border-radius: 8px;
padding: 12px;
font-size: 14px;
color: #374151;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}

QLineEdit:focus{
border: 2px solid #7f8082;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ex: √ìleo de motor</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Descri√ß√£o *</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>200</y>
         <width>311</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit[echoMode=&quot;2&quot;], QLineEdit[echoMode=&quot;0&quot;] {
                border: 2px solid #e5e7eb;
                border-radius: 8px;
                padding: 12px;
                font-size: 14px;
                color: #374151;
            }
QLineEdit:focus{
border: 2px solid #7f8082;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ex: 24.50</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>360</y>
         <width>150</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: transparent;
color: Black;
border: 2px solid;
border-radius: 8px;
padding: 12px;
font-size: 14px;
font-weight: bold;
min-height: 20px;
}

QPushButton:hover{
background-color: rgb(234, 227, 174);
}

QPushButton:Pressed{
background-color: rgb(210, 203, 150);
}</string>
       </property>
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>170</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Quantidade em estoque *</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>55</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>C√≥digo produto *</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>80</y>
         <width>341</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
border: 2px solid #e5e7eb;
border-radius: 8px;
padding: 12px;
font-size: 14px;
color: #374151;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}

QLineEdit:focus{
border: 2px solid #7f8082;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ex: √ìleo de motor</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>200</y>
         <width>341</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit[echoMode=&quot;2&quot;], QLineEdit[echoMode=&quot;0&quot;] {
                border: 2px solid #e5e7eb;
                border-radius: 8px;
                padding: 12px;
                font-size: 14px;
                color: #374151;
            }
QLineEdit:focus{
border: 2px solid #7f8082;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ex: 50</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1298</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
